string robot_hw_switch_ems_status
string robot_hw_switch_key_status
string robot_hw_switch_reset_status
string robot_hw_switch_start_stop_status
string robot_is_charging_status
string robot_is_sensor_stable_depth_front
string robot_is_sensor_stable_depth_rear
string robot_is_sensor_stable_imu
string robot_is_sensor_stable_lidar_3d
string robot_is_sensor_stable_lidar_front
string robot_is_sensor_stable_lidar_rear
string robot_is_sensor_stable_motor
string robot_is_sensor_stable_tof